src/DevilDaggersInfo.Tools/Ui/Practice/Main/InputValuesChild.cs

```diff
@@ -33,7 +33,7 @@ public static void Render()
 			ImGui.Spacing();
 			ImGuiImage.Image(gemOrHomingTexture.Id, new Vector2(16), tintColor);
 			ImGui.SameLine();
-			ImGui.InputInt("Added gems", ref PracticeLogic.State.AdditionalGems, 1);
+			ImGui.InputInt((PracticeLogic.State.HandLevel is HandLevel.Level2)? Inline.Span($"Added gems({PracticeLogic.State.AdditionalGems+10})") : "Added gems", ref PracticeLogic.State.AdditionalGems, 1);
```

Comment:
Could you open an issue for this? I'm thinking there's probably a better way to make this more clear. I'd also prefer commits/PRs with a single purpose.

Also I'll be refactoring all of the UI soon, so I'm postponing UI changes until then because it'll be a big change.